packages/actions-global/src/types.ts

```diff
@@ -8,25 +7,23 @@ import type {
   TAppNotificationValuesUnexpectedError,
 } from '@commercetools-frontend/constants';
 
-export type TShowNotification = TNotification & {
```

Comment:
Removing this merge fixes the issue mentioned in the PR description, regarding the `id`.

<img width="700" alt="image" src="https://user-images.githubusercontent.com/1110551/157537568-9d1590ad-4292-4e16-8db3-a1e1bf21cd8b.png">